c10/util/string_view.h

```diff
@@ -588,8 +595,12 @@ constexpr inline void swap(
     basic_string_view<CharT>& rhs) noexcept {
   lhs.swap(rhs);
 }
-
+using string_view_ext = basic_string_view<char>;
+#ifdef fbcode_caffe2
```

Comment:
This should have been uppercase :)
```suggestion
#ifdef FBCODE_CAFFE2
```